internal/scripts/installation/modulefiles/5.1.0_orion

```diff
@@ -0,0 +1,25 @@
+#%Module######################################################################
+##
+##      METplus
+##
+proc ModulesHelp { } {
+  puts stderr "Sets up the paths and environment variables to use the METplus-5.1.0
+     *** For help see the official MET webpage at http://www.dtcenter.org/met/users ***"
+}
+#%Module######################################################################
+##
+##      METplus
+##
+proc ModulesHelp { } {
+  puts stderr "Sets up the paths and environment variables to use the METplus-5.1.0
+     *** For help see the official MET webpage at http://www.dtcenter.org/met/users ***"
+}
+
+module load contrib
+module load intel/2020.2
+module load met/11.1.0-beta1
```

Comment:
Should the met module load, METPLUS_PATH, and PATH values include beta1?

Reply:
@georgemccabe It doesn't really matter, as it's mainly just a backup and for me, but I do usually remove that to be more generic.  I just made the change to remove it.  Thanks for noticing and mentioning it.